failed to build module on centos:7

```
[ 63%] Building C object src/CMakeFiles/merger.dir/compat/utils.c.o
[ 72%] Building C object src/CMakeFiles/merger.dir/merger/merger.c.o
/tmp/luarocks_tuple-merger-0.0.1-1-Cwe2jE/tuple-merger/src/merger/merger.c: In function 'luaT_merger_new_parse_sources':
/tmp/luarocks_tuple-merger-0.0.1-1-Cwe2jE/tuple-merger/src/merger/merger.c:306:2: error: 'for' loop initial declarations are only allowed in C99 mode
  for (uint32_t i = 0; i < source_count; ++i) {
  ^
/tmp/luarocks_tuple-merger-0.0.1-1-Cwe2jE/tuple-merger/src/merger/merger.c:306:2: note: use option -std=c99 or -std=gnu99 to compile your code
gmake[5]: *** [src/CMakeFiles/merger.dir/merger/merger.c.o] Error 1
gmake[4]: *** [src/CMakeFiles/merger.dir/all] Error 2
gmake[3]: *** [all] Error 2
```

--- a/.github/workflows/luarocks-build.yml
+++ b/.github/workflows/luarocks-build.yml
@@ -65,7 +65,7 @@ jobs:
         run: |
           cd $GITHUB_WORKSPACE/tarantool/build
           cmake --build . --config $BUILD_TYPE --target install -- -j
-          echo "::set-env name=PATH::$GITHUB_WORKSPACE/share/bin:$PATH"
+          echo "PATH=$GITHUB_WORKSPACE/share/bin:$PATH" >> $GITHUB_ENV
 
 
       - name: luarocks make and pack
@@ -92,7 +92,7 @@ jobs:
         shell: bash
         run: |
           brew install make
-          echo "::set-env name=PATH::$(brew --prefix)/opt/make/libexec/gnubin:$PATH"
+          echo "PATH=$(brew --prefix)/opt/make/libexec/gnubin:$PATH" >> $GITHUB_ENV
 
       - name: check examples
         run: |

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,6 +11,9 @@ set(CMAKE_MODULE_PATH
 
 include(cmake/utils.cmake)
 
+# Set CFLAGS/CXXFLAGS
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Werror -std=gnu99")
+
 # Find Tarantool dependecies
 set(TARANTOOL_FIND_REQUIRED ON)
 find_package(Tarantool)

